platform/brains2/src/lib/custom_teensy_audio/output_pt8211.h

```diff
@@ -52,6 +52,7 @@ class AudioOutputPT8211 : public AudioStream
 	AudioOutputPT8211(void) : AudioStream(2, inputQueueArray) { begin(); }
```

Comment:
I think we should remove `begin` from the constructor, and call it exlicitly in `main`, so we have well-defined initialization order, rather than relying  on the order of static initialization.